src/unit-tests/inc/ut_os_support.h

```diff
@@ -79,7 +79,7 @@ static inline bool UtOsalImplemented(int32 Fn, const char *File, uint32 Line)
 /*--------------------------------------------------------------------------------*/
 
 #define UT_os_sprintf(buf,...)  \
-    snprintf(buf,sizeof(buf),__VA_ARGS__)
+    do { int x = snprintf(buf,sizeof(buf),__VA_ARGS__); if (x > 0) buf[x] = 0; } while (0)
```

Comment:
Was this line due an update?

Reply:
Yeah, needed an upper bound check, see 6175171